views/fe-utils/fetchUtil.js

```diff
@@ -140,6 +140,34 @@ const createChart = async (formValues) => {
     return res.json()
 }
 
+const getCharts = async () => {
+  const res = await window.fetch(`${routes.basePath}/api/v1/charts`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    credentials: 'same-origin',
+  })
+
+  if (res.status !== 200) return new Error(res.message)
+
+  return res.json()
+}
+
+const deleteChart = async (id) => {
+  const res = await window.fetch(`${routes.basePath}/api/v1/charts/${id}`, {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    credentials: 'same-origin'
+  })
```

Comment:
What do you think about making these route paths part of the routes object? That way if the path needs to change we only need to update the constant rather than each function that hits that route

Just an observation, It seems like most of our api calls use these same options (with different `method`s). 🎱 _(optional)_ it might be worth making something generic that sets that up for us once, ( something like `API.get(<path>)`) rather than having each API function have to set up the same headers. We could also have specific API objects for different resources, like a ChartsApi that uses the generic API methods. Again super optional, just something to think about as we add more functions like this. 🎱 

A lighter weight alternative could be to pull some of these shared options to a constant that can be spread.

```js
// <path>/routes/constants.js
const basePath = 'a-good-host.com'

export const routes = {
  // ... existing routes
  charts: `${basePath}/api/v1/charts`,
  chart: (id) => `${basePath}/api/v1/charts/${id}`
}

export const defaultApiOptions = {
  headers: {
    'Content-Type': 'application/json'
  },
  credentials: 'same-origin'
}

// this file
import { routes, defaultApiOptions } from '<path>/routes/constants.js'

// ...

const deleteChart = async (id) => {
  const res = await window.fetch(
    routes.chart(id),
    { ...defaultApiOptions, method: 'DELETE' }
  )

  if (res.status !== 200) return new Error(res.message)

  return res.json()
}
```